ddl/index.go

```diff
@@ -175,6 +176,23 @@ func dropIndexColumnFlag(tblInfo *model.TableInfo, indexInfo *model.IndexInfo) {
 	}
 }
 
+func validateRenameIndex(from, to model.CIStr, tbl *model.TableInfo) (duplicate bool, err error) {
```

Comment:
`duplicate` is not a proper name for the first return value. It confused me. I thought it is the case that the to.L is the same as any other index name.

Reply:
`ignore` is better

Reply:
reasonable. I've fixed this. :)